### ISSUE TITLE
Configure.ac checks for SVN

### ISSUE THREAD

**Issue by elextr** [geany/geany#856]
```
# check for SVN revision
GEANY_CHECK_REVISION([dnl force debug mode for a SVN working copy
					  CFLAGS="-g -DGEANY_DEBUG $CFLAGS"])
```

Hopefully its just the comments are wrong and it actually checks git, but someone who knows where to look should check.

**Comment by codebrainz** [comment]
Macro is defined here:

https://github.com/geany/geany/blob/master/m4/geany-revision.m4

**Comment by elextr** [comment]
That comments about SVN too, but to my non-m4 ejurkated mind does not seem to test for it.  Is that right?

**Comment by codebrainz** [comment]
The comments must've been missed in 69313263e7bde140eba16a87535beb089e4f97c1. I suspect at some point they were less confusing, during the transition from SVN to Git, when git-svn was being used.

**Comment by b4n** [comment]
As @codebrainz pointed out, it's a leftover of the Git migration, the actual check is correct.  I fixed this in 8099fddd9262be2888986a40e842b78f13dac050
